.github/workflows/ci-cd.yml

```diff
@@ -77,12 +77,15 @@ jobs:
       uses: astral-sh/setup-uv@v5
       with:
         python-version: ${{ matrix.python-version }}
+        enable-cache: true
```

Comment:
I believe this is unnecessary, because the setup-uv caches by default since https://github.com/astral-sh/setup-uv/releases/tag/v5.0.0. Also the default glob should be sufficient for our case.

Reply:
cool, reverted, thanks

Reply:
they should update the docs to stay the default is true, would have prevented me from going down this path